examples/guides/animation/step-1.html

```diff
@@ -3,19 +3,13 @@
 
 <head>
     <!-- Include CARTO VL JS -->
-    <script src="../../../../dist/carto-vl.js"></script>
+    <script src="../../../dist/carto-vl.js"></script>
```

Comment:
It's fine to change this to adapt to local carto-vl path. At dev-center should be automatically replaced by the corresponding version